artifactory_cleanup/cli.py

```diff
@@ -166,9 +166,17 @@ def main(self):
             print(str(err), file=sys.stderr)
             sys.exit(1)
 
-        server, user, password = loader.get_connection()
+        server, user, password, apikey = loader.get_connection()
         session = BaseUrlSession(server)
-        session.auth = HTTPBasicAuth(user, password)
+        if apikey != "":
+            print("Using API Key")
+            headers = {
+                "X-JFrog-Art-Api": apikey
+            }
+            session.headers = headers
+        else:
+            print("Using user and password")
+            session.auth = HTTPBasicAuth(user, password)
```

Comment:
Let's simplify it a bit
```suggestion
        if apikey:
            print("Using API Key")
            headers = {
                "X-JFrog-Art-Api": apikey
            }
            session.headers = headers
        else:
            print("Using user and password")
            session.auth = HTTPBasicAuth(user, password)
```